hieradata/class/staging/jenkins.yaml

```diff
@@ -13,7 +13,6 @@ govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::jobs::smokey
   - govuk_jenkins::jobs::smokey_deploy
   - govuk_jenkins::jobs::update_cdn_dictionaries
-  - govuk_jenkins::jobs::user_monitor
```

Comment:
It looks like you're not adding it back into staging on AWS - is that intended?

Reply:
It's [already there](https://github.com/alphagov/govuk-puppet/blob/f8b97320c9101ba25de39242f6b68aa7f95a7b7a/hieradata_aws/class/staging/jenkins.yaml#L82).

Reply:
Oh, silly me 🤦  Sorry for the noise.